runtime/queries/just/textobjects.scm

```diff
@@ -0,0 +1,46 @@
+(body) @function.inside
+(recipe) @function.around
+(expression 
+    if:(expression) @function.inside 
+) 
+(expression 
+    else:(expression) @function.inside
+) 
+(interpolation (expression) @function.inside) @function.around
+(settinglist (stringlist) @function.inside) @function.around
+
+(call (NAME) @class.inside) @class.around
+(dependency (NAME) @class.inside) @class.around
+(depcall (NAME) @class.inside)
+
+(dependency) @parameter.around
+(depcall) @parameter.inside
+(depcall (expression) @parameter.inside) 
+
+(stringlist 
+    (string) @parameter.inside
+    . ","? @_end
+    (#make-range! "parameter.around" @parameter.inside @_end)
+)
+(parameters 
+    [(parameter) 
+    (variadic_parameters)] @parameter.inside
+    . " "? @_end
+    (#make-range! "parameter.around" @parameter.inside @_end)
+)
```

Comment:
I think the `make-range!` property is only implemented in nvim. For now we may be only able to capture `parameter.inside` since I don't think we have anything that would work the same